packages/file_selector/file_selector_ios/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: file_selector_ios
+description: iOS implementation of the file_selector plugin.
+repository: https://github.com/flutter/plugins/tree/master/packages/file_selector/file_selector_ios
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+file_selector%22
+version: 0.5.0
+
+environment:
+  sdk: ">=2.14.4 <3.0.0"
+  flutter: ">=2.0.0"
+
+flutter:
+  plugin:
+    implements: file_selector
+    platforms:
+      ios:
+        dartPluginClass: FileSelectorIOS
+        pluginClass: FTLFileSelectorPlugin
+
+dependencies:
+  cross_file: ^0.3.1
```

Comment:
Do you need this dependency? f_s_p_i should be handling that.

Reply:
Done.